### ISSUE TITLE
Focus outline on wrong element

### ISSUE THREAD

**Issue by xougat** [znsio/engioscope#60]
**Describe the bug**
Focus outline on button is on the icon instead for the filters - affects sort by and  multi filter drawer icon near the search 🔎 field

**To Reproduce**
1. Go to 'repositories'
2. Click on 'filter button' beside search field
3. If focus outline is shown, behavior can be noticed. 

**Expected behavior**
Focus outline should be on the button, not icon
